keps/sig-network/20191227-app-protocol.md

```diff
@@ -99,11 +100,19 @@ It may take some time for cloud providers and other consumers of these APIs to
 support this attribute. To help with this, we will work to communicate this
 change well in advance of release so it can be well supported initially.
 
+### Proposed Roadmap
+**Kubernetes 1.18**: New field is added but gated behind new alpha
+`ServiceAppProtocol` feature gate.
+**Kubernetes 1.19**: `ServiceAppProtocol` feature gate graduates to beta and is
```

Comment:
What feedback do we think we might get in beta that we have not already heard?  IOW, convince me that it's safe to skip.

Reply:
The rationale was just to follow the standard procedure, although this likely won't change anything for end users. It does feel like overkill to have alpha->beta->GA for a new field like this, but it also doesn't add too much work and seems to be the right way to add new functionality to Kubernetes. This was originally based on a conversation with @liggitt, does this match what you were thinking here?